ansible/group_vars/all.yml

```diff
@@ -9,7 +9,7 @@ encrypted_root: "/opt/data"
 
 shared_drive_enabled: true  # overridden in ../config/$ENV/$ENV.yml
 shared_dir_gid: 1500  # This GID cannot already be allocated
-shared_dir_name: "shared_data{{ '_' ~ deploy_env if deploy_env != 'production' else '' }}"
+shared_dir_name: "shared{{ '_' ~ deploy_env if deploy_env != 'production' else '' }}"
```

Comment:
i changed the name of the shared drive to create a new mountpoint for the device. if the old directory name is preferred, we can switch it back once the new drive is set up.